swri_roscpp/include/swri_roscpp/dynamic_parameters.h

```diff
@@ -392,6 +426,34 @@ namespace swri
         desc.level = 0;
         desc.description = param->second.description;
         desc.edit_method = "";
+
+        // If this is an enum, lets make the edit method string
+        for (int i = 0; i < param->second.enums.size(); i++)
+        {
+          if (i == 0)
+          {
+            desc.edit_method += "{'enum_description': '" + escapeString(desc.description) + "'";
```

Comment:
Assembling YAML strings manually seems dangerous, especially writing our own method to escape strings.  Why not use yaml-cpp?

Reply:
Was going for the most prudent solution and didn't want to add extra dependencies.